### PR TITLE
Add support for preparing Deal Data

### PR DESCRIPTION
**Pull request by ronykris**
Hi,

I have created a module in storage SDK to process the user specified file and fetch the data required for deal making in FVM. Below is the sample code to invoke the getPrepData() module.

async function main () {
    const dealData = await client.getPrepData('README.md')
    console.log(dealData)
}

main()
If this PR goes through I will create another PR to update the docs. Please let me know if you have questions.

Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,16 @@
   </picture>
 </p>
 
+<h1 align="center">Storage SDK</h1>
```
Comment (rekpero) [inline README.md:9]: Can you revert this file change?

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sdk",
-  "version": "0.0.0",
+  "version": "0.0.1",
```
Comment (rekpero) [inline package.json:3]: no need to change this, as we are not changing the whole SDK version

**Review by rekpero — CHANGES_REQUESTED** [review]
Please don't change any readme or content written by the team.

File: package.json (hunk 2 above)
Comment (ronykris) [reply package.json]: fixed it.

File: README.md (hunk 1 above)
Comment (ronykris) [reply README.md:9]: Reverted, this happened inadvertently. I am sorry!

Comment (rekpero) [reply README.md]: no worries

**Comment by ronykris** [comment]
The change in the README was inadvertent. I am sorry I did not realise it.

**Comment by rekpero** [comment]
No worries, PR is quite good. Let @shomix take a look.

**Pull request closed without merging**